src/ivoc/ivocvect.cpp

```diff
@@ -1570,23 +1570,21 @@ static Object** v_copy(void* v) {
 
 
 static Object** v_at(void* v) {
-    Vect* x = (Vect*) v;
-
-    int top = x->size() - 1;
```

Comment:
This triggered warnings about wrap-around when size (which is unsigned) is zero.